homeassistant/components/coronavirus/strings.json

```diff
@@ -6,6 +6,8 @@
         "data": { "country": "Country" }
       }
     },
-    "abort": { "already_configured": "This country is already configured." }
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%] for this country."
```

Comment:
Maybe we can just remove `for this country`?

Reply:
Done